hamilton/plugins/pandas_extensions.py

```diff
@@ -229,6 +229,28 @@ def name(cls) -> str:
         return "pickle"
 
 
+@dataclasses.dataclass
+class PandasJsonLoader(DataLoader):
+    """Data loader for JSON files using Pandas. Note that this currently does not support the wide array of
+    data loading params (e.g., precise_float, encoding). We will be adding this in over time, but for now
+    you can subclass this or open up an issue if this doesn't have what you want."""
+
+    path: str
+
```

Comment:
we'll want this part to reflect all the different parameters you can pass -- https://pandas.pydata.org/docs/reference/api/pandas.read_json.html#pandas-read-json

So have a look at https://github.com/DAGWorks-Inc/hamilton/blob/main/hamilton/plugins/pandas_extensions.py#L153-L229 for example.